packages/core/src/command.ts

```diff
@@ -61,14 +61,16 @@ class Command {
             } else {
               cmdStr += ','
             }
-
-            cmdStr += `${key}=${escapeProperty(val)}`
+            // safely escape the property - avoid blowing up when attempting to
```

Comment:
It looks like these lines should be reverted. The functions `escapeData` and `escapeProperty` already OR the value with empty string

Reply:
Those functions only || with an empty string if the value passed is falsey. Booleans, ints, and objects with no `.replace` function crash when passed into those functions because `123 || ""` will return 123.